scripts/dotnet.py

```diff
@@ -480,23 +493,22 @@ def install(
 
     # If Version is supplied, pull down the specified version
 
-    cmdline_args = dotnetInstallInterpreter + [
-            '-InstallDir', install_dir,
-            '-Architecture', architecture
+    common_cmdline_args = dotnetInstallInterpreter + [
+        '-InstallDir', install_dir,
+        '-Architecture', architecture
     ]
-    if version is not None:
-        cmdline_args = cmdline_args + [
-            '-Version', version,
-        ]
-        RunCommand(cmdline_args, verbose=verbose).run(
-            get_repo_root_path()
-        )
-    else:
-        # Install Runtime/SDKs
+
+    # Install Runtime/SDKs
+    if versions:
+        for version in versions:
+            cmdline_args = common_cmdline_args + ['-Version', version]
+            RunCommand(cmdline_args, verbose=verbose).run(
+                get_repo_root_path()
+            )
+
+    if channels:
```

Comment:
what prevents the user from specifying version and channels? does that make sense?

Reply:
Yes, it makes sense. You are requesting to download/install the different SDKs you want to use to build, benchmark, and compare results. For example, you might want to compare LTS vs netcore3.0-preview4 vs netcoreapp3.0-preview5. This enable the facility to do that. It's a simple fix for a bad concatenation of the dotnet-install command line, and a simple improvement to the version feature.